### PR TITLE
Fix failing test build and add CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by juwara0**
**This project uses [semver](http://semver.org), please check the scope of this pr:**

- [X] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG
**Added** a CONTRIBUTING.md file
**Updated** ember version to 2.11.2 to address failing tests from [BUGFIX](https://github.com/emberjs/ember-mocha/issues/141)

### CODE REVIEW

File: config/coverage.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 module.exports = {
   coverageEnvVar: 'COVERAGE',
   coverageFolder: 'coverage',
+  excludes: [
+    /app\/(.*)/,
```
Comment (sandersky) [inline config/coverage.js]: What is the purpose of ignoring app?

File: config/coverage.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+    /dummy\/(.*)/
```
Comment (sandersky) [inline config/coverage.js]: I actually don't want to exclude dummy as eventually I plan to add acceptance tests that ensure the dummy app continues to work properly.

Comment (juwara0) [inline config/coverage.js]: Reverting per ^

**Comment by sandersky** [comment]
:+1:

**Pull request merged**